proxy/http2/HTTP2.cc

```diff
@@ -665,11 +666,19 @@ http2_decode_header_blocks(HTTPHdr *hdr, const uint8_t *buf_start, const uint8_t
       }
     }
 
+    // turn on that we have a trailer header
+    char trailer_name[] = "trailer";
+    if (name_len == (sizeof(trailer_name) - 1) && strncmp(name, trailer_name, sizeof(trailer_name) - 1) == 0) {
+      trailing_header = true;
```

Comment:
`is_trailing_header` ? It looks like `trailing_header` is not used in below code.